packages/react-app/pages/profile/approval/index.js

```diff
@@ -89,21 +112,18 @@ function ApproveShareContactInformation() {
                     right: 3,
                   }}
                 />
+                <br />
                 <Badge px={2} py={1} bg={useColorModeValue("gray.50", "gray.800")} fontWeight={"400"}>
-                  {requesterAddress}
+                  {recruiter.address.slice(0, 10)}
```

Comment:
Ideally, we would want to see the whole address. Or atleast the common: `0xAAA...AAA` pattern.